src/c++/perf_analyzer/genai-perf/genai_perf/wrapper.py

```diff
@@ -76,36 +76,43 @@ def build_cmd(args, extra_args):
 
         utils.remove_file(args.profile_export_file)
 
-        cmd = f"perf_analyzer -m {args.model} --async --input-data {DEFAULT_INPUT_DATA_JSON} "
+        cmd = [
+            f"perf_analyzer",
```

Comment:
```suggestion
            "perf_analyzer",
```

Reply:
NO COMMENTS ALLOWED

Reply:
why strip the f string?

Reply:
It's unnecessary to use an f-string since there are no replacement fields in this case.

https://docs.python.org/3/reference/lexical_analysis.html#f-strings

Reply:
Since this is not perf critical, i rather keep the code consistent.

Reply:
Thanks for enumerating these. Lets update those too!

Reply:
Ok, maybe f-strings here is ok cuz switching between them could be harder to read.

Reply:
Discussed offline.
Takeaways: 
- shooting for consistency in the code block. 
- avoid f strings in logging statements and perf crit code, due to the extra func calls
- continue to try and refine the code as we go along

Reply:
Yeah sounds good. I think using f-string altho unnecessary is fine here because it follows a local pattern, but in general, in the rest of the codebase, where f-strings aren't interwoven, normal strings are fine.

Reply:
@nv-hwoo are you ok updating the other non-f-string cases in this list to use f-string for consistency?

Reply:
@matthewkotila whoops, sorry didn't see the discussion here. Will make the change.